tests/login/001/signup_1b.spec.js

```diff
@@ -0,0 +1,184 @@
+// /*
+// testing Funnel 1B's Log-In/Sign-Up page
+//
+// FIRST PAGE FOR FUNNEL 1B: https://www.kidguard.com/funnel/form/001/b?lander=home&
+// Description: Including "first name," "last name," "email," "phone," "state,"
+// and "zip code"
+//
+// SECOND PAGE FOR FUNNEL 1B
+// includes passowrd,verify password, and security question/answer
+// */
+
+"use strict";
+var myModule = require('../login.library.js');
+
+const form = { 
+                  "name": "input[name='first_name']",
+                  "last": 'input[name="last_name"]',
+                  "zipcode": "input[id='zipcode']", 
+                  "email": 'input[name="email"]',
+                  "phone": 'input[name="phone"]',
+                  "exp": "0", 
+                  "expmonth": "input[name='expiration_month']", 
+                  "expyear": "input[name='expiration_year']",
+                  "zipcode": 'input[id="zipcode"]',
+                  "password": 'input[id="password"]', 
+                  "verify": 'input[id="password_verify"]',
+                  "secret_q": 'select[id="secret_question"] option:contains("color")',
+                  "secret_answer": 'input[id="secret_question_answer"]',
+                  "password_submit": 'button.submit_button'
+
+}
+
+const Nightmare = require( "nightmare" ),
+      expect = require( "chai" ).expect,
+      BASE_URL = "https://www.kidguard.com/funnel/form/001/b?lander=home&",
+      onError = ( err ) => {
+        console.error( "Test-runner failed:", err );
+      },
+      browser = new Nightmare({
```

Comment:
one possible refactoring point is extracting nightmare initialization to a module then require it into your spec file.
it's called factory pattern.

```
// nightmare-factory.js
const Nightmare = require('nightmare');

modules.exports.create = function() {
  return new Nightmare({ ... });
}
```

```
// your spec.js
const factory = require('./nightmare-factory');

const browser = factory.create();
...
```